slick/src/sphinx/developers.rst

```diff
@@ -0,0 +1,155 @@
+For Slick developers
```

Comment:
For consistency with the rest of the site, that should be a capital D in "Developers".